saltapi/exceptions_handling.py

```diff
@@ -50,7 +50,7 @@ async def validation_exception_handler(
         )
 
     @app.exception_handler(Exception)
-    async def exception_handle(request: Request, exc: Exception) -> Response:
+    async def exception_handle(request: Request) -> Response:
```

Comment:
Maybe rather call this something like `generic_exception_handler`, given its a handler rather than handle?

Reply:
"handler", not "handle"...